cockroachcloud/export-logs.md

```diff
@@ -291,11 +407,11 @@ Where:
 
 ### Is it possible to configure exported logs to be redacted at source?
 
-Logs exported in this fashion retain [`redactable`](/docs/{{site.versions["stable"]}}/configure-logs.html#redact-logs) markers, but are **not** themselves redacted. If you need to redact sensitive log data, you can use these `redactable` markers to do so once log entries have been written to your configured cloud log sink.
+Yes, use the `redact: true` log configuration option. See [Redact logs](/docs/{{site.versions["stable"]}}/configure-logs.html#redact-logs) for more information.
 
-### Is it possible to configure multiple log export configurations to send different log channels to different log groups in my cloud log sink?
+### Is it possible to send different log channels to different log groups in my cloud log sink?
 
-No, only one log export configuration is currently possible per cluster.
+Yes, use the custom log configuration step for your cloud provider, and specify multiple `groups`, each with a unique `name` value, in your configuration.
```

Comment:
For consistency, can we say "unique `log_name` value" instead of "unique `name` value"?